exonum-java-binding-core/pom.xml

```diff
@@ -275,6 +291,13 @@
       <scope>provided</scope>
     </dependency>
 
+    <!--It is needed for Javadoc generation with JDK 9-->
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>${javax-annotation-api.version}</version>
```

Comment:
Is it needed in the default scope? Will `provided` scope fix the problem, so it is not packaged anywhere?

Asking because it is GPL, we cannot include it (but can migrate probably to better alternatives).